requirements/env_omnisci.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - pytest-xdist>=2.1.0
   - coverage<5.0
   - pygithub==1.53
-  - pyomniscidbe<=5.7.1 # modin-project/modin#3574
+  - pyomniscidbe
```

Comment:
I'm thinking probably about <=5.8.* in order to avoid issues like we had after 5.8...

Reply:
done, used `<=5.8` instead of `<=5.8.*` since conda treating them the same but deprecated the second one.

<details><summary>Warning message</summary>

```
WARNING conda.models.version:get_matcher(540): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 5.8.*, but conda is ignoring the .* and treating it as 5.8
```

</details>

Reply:
@Garra1980 , What issues are you saying about?